appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/model/Series.java

```diff
@@ -51,13 +51,14 @@
  */
 public final class Series implements Comparable<Series>, Serializable {
 
+    private static final char QUERY_WILDCARD = '*';
     public static final char TAG_ASSIGN = ':';
     public static final char TAG_SEPARATOR = ' ';
     private static final char[] TAG_SEPARATORS = { ' ', ',', ';' };
 
     private static final String SPLIT_PATTERN = "[" + Pattern.quote(new String(TAG_SEPARATORS)) + "]+";
 
-    private final String metric;
```

Comment:
Getter can be defined for the `metrics` variable as defined for other variables in the immutable `Series` class.